builtins/amp-img.js

```diff
@@ -167,6 +169,17 @@ export class AmpImg extends BaseElement {
     return true;
   }
 
+  /** @override */
+  firstLayoutCompleted() {
+    const placeholder = this.getPlaceholder();
+    if (isExperimentOn(this.win, 'blurry-placeholder') && placeholder && 
```

Comment:
Tiny improvement: `isExperimentOn` could read the cookies, which is an operation that's a bit more expensive than checking a class. Maybe it'd be worth moving the `isExperimentOn()` check last?